### PR TITLE
Ft admin jobs page

### PR DESCRIPTION
**Pull request by neelxie**
#### What does this PR do?

Display Jobs List

#### Description of Task to be completed?
- List Cluster jobs in the admin dash

#### How should this be manually tested?

* Check it out here.

### CODE REVIEW

**Comment by Mubangizi** [comment]
@neelxie  **Jobs** Have a `duration` and `completions` count.
`Duration` Shows the time the Job took while `Completions` Count Shows how many tasks its **finished**.
Please add those columns Thank you

**Comment by MutegekiHenry** [comment]
> @neelxie **Jobs** Have a `duration` and `completions` count.
> `Duration` Shows the time the Job took while `Completions` Count Shows how many tasks its **finished**.
> Please add those columns Thank you

These fields are relevant

**Comment by neelxie** [comment]
> Please add _Completions_ and _Duration_ columns thank you @neelxie

@Mubangizi and @MutegekiHenry kindly do point out the difference between the Age attribute that is already available, and duration that you are all suggesting? I have added the durations as suggested too, though it just doesnt make sense to me.

**Comment by StephenAraka** [comment]
> > Please add _Completions_ and _Duration_ columns thank you @neelxie
> 
> @Mubangizi and @MutegekiHenry kindly do point out the difference between the Age attribute that is already available, and duration that you are all suggesting? I have added the durations as suggested too, though it just doesnt make sense to me.

Yeah, I also feel like it sounds like repetition. Unless they are really necessary. @Mubangizi  are all  the columns necessary?

**Comment by neelxie** [comment]
> Please add _Completions_ and _Duration_ columns thank you @neelxie

@Mubangizi here are your changes
image

**Comment by Mubangizi** [comment]
> > Please add _Completions_ and _Duration_ columns thank you @neelxie
> 
> @Mubangizi here are your changes
> image

How okay let me turn a bling eye for the completions bit

File: src/redux/reducers/jobsReducer.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+
+import {
+  FETCH_JOBS_SUCCESS,
+  FETCH_JOBS_FAILED,
+  IS_FETCHING
```
Comment (Mubangizi) [inline src/redux/reducers/jobsReducer.js]: @neelxie please separate the `IS_FETCHING` action for Namespaces and Jobs. Make them different.
I inquired about this to detail and i was told by @ckwagaba that each component should have its own `ACTION` no-matter how similar they look

Comment (neelxie) [reply src/redux/reducers/jobsReducer.js]: @Mubangizi have a full understanding of what is happening and how it is being used, and please kindly embrace re-usability, I fully understood @ckwagaba point, you didnt.

Comment (Mubangizi) [reply src/redux/reducers/jobsReducer.js]: I also wanted to do the same thing. But i was told not to @neexie

Comment (neelxie) [reply src/redux/reducers/jobsReducer.js]: @Mubangizi unless you have a detailed elaboration of how that affects performance other than hearsay, I know codebases that have this in use and the name is @neelxie not @neexie

**Pull request merged**